static/src/javascripts-legacy/projects/common/views/svgs.js

```diff
@@ -145,10 +143,11 @@ define([
     };
 
     return function (name, classes, title) {
-        if (config.switches.webpack) {
+        // #wp-rjs - needed by karma
+        if (svgs[name].markup) { // webpack way
```

Comment:
Indeed it is. In fact you don't even need `svg` if you don't want to add classes or a title to the SVG.